app/src/organisms/AppSettings/ConnectRobotSlideout.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import {
+  Flex,
+  ALIGN_FLEX_END,
+  DIRECTION_COLUMN,
+  SPACING,
+  Text,
+  TYPOGRAPHY,
+  COLORS,
+  Icon,
+  SIZE_2,
+  Link,
+} from '@opentrons/components'
+import { ManualIpHostnameForm } from './ManualIpHostnameForm'
+import { IpHostnameList } from './IpHostnameList'
+import { Slideout } from '../../atoms/Slideout'
+import { PrimaryButton } from '../../atoms/Buttons'
+import { ExternalLink } from '../../atoms/Link/ExternalLink'
+import { Divider } from '../../atoms/structure'
+import { StyledText } from '../../atoms/text'
+import { getScanning, startDiscovery } from '../../redux/discovery'
+
+import type { Dispatch, State } from '../../redux/types'
+
+const SUPPORT_PAGE_LINK =
+  'https://support.opentrons.com/en/articles/2934336-manually-adding-a-robot-s-ip-address'
+
+interface ConnectRobotSlideoutProps {
```

Comment:
You should export this type so you can import it in your test and cast your props to this type to get rid of the type errors that are failing in CI
```suggestion
export interface ConnectRobotSlideoutProps {
```